src/server/game/Entities/Item/Container/Bag.h

```diff
@@ -19,6 +19,7 @@
 #define ACORE_BAG_H
 
 // Maximum 36 Slots ((CONTAINER_END - CONTAINER_FIELD_SLOT_1)/2
+#include <memory>
```

Comment:
i wouldn't move to all includes, because that would slow down compile time, would increase binary size and i think that make non necessary header usage

Reply:
Edit: i missunderstood a little, i will move inclusion above code comment. At base i thought you mean to move to global header (like pch header or similar)